.eslintrc.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  parser: 'babel-eslint',
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'preact',
+    'plugin:flowtype/recommended',
+    'plugin:prettier/recommended',
+    'prettier/standard',
+  ],
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+    mocha: true,
+    'jest/globals': true,
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      modules: true,
+      jsx: true,
+    },
+  },
+  plugins: ['react', 'flowtype', 'mocha', 'prettier'],
+  globals: {
+    expect: false,
+  },
+  rules: {
+    /**
+     * Temporary disabled rules for CI passing
+     * @TODO merge improvement/linters-fixes PR
```

Comment:
@kopijunkie inline documents like this

Reply:
Ah, yes. Good point about inline documentation, since JSON doesn't allow for that.